content/ny-funksjonalitet/roadmap/autorisasjon/_index.md

```diff
@@ -89,6 +89,11 @@ Derfor ønsker vi å vise brukers rettigheter når hun slår opp et tjeneste i s
 For å reduesere risiko har Altinn i en periode hatt to løsnigner for å lese data fra folkeregisteret, gammel DSF-integrasjon og  integrasjon mot nytt FREG. 
 For å sikre effektiv forvaltning og drift av autorisasjonsløsningen skal vi konsolidere og rydde opp i kode som ikke lenger er i bruk knyttet til gammel DSF-integrasjon. 
 
+### 18 - Nedlastbar oversikt over alle delegeringer som er gjort for en virksomhet og alle dens underenheter (Q2 - 2021)
+For å sikre tilgangsstyrer for virksomheter med flere underenheter en bedre oversikt på tvers tilbys en funksjonlatiet for å laste ned oversikt over retiigheter som finnes. 
+Det vil være mulig å hente ut oversikt for hvoedenhet og tilhørende underenheter. 
+![Last ned fil ](skjermbilde_22793.PNG "Last ned fil med oversikt over rettigheter")
```

Comment:
- Fjern mellomrom i [Last ned fil` `]
- Endre filnavn til noe beskrivende, f.eks. `last-ned-fil-med-oversikt-over-rettigheter.png`.
  - Bruk `-`, ikke underscore
  - Ikke bruk uppercase `PNG` (dette er en bug i verktøyet du har brukt for å ta screenshot)

Reply:
Screenshot har har også dårlig kvalitet (blurry), og bør fokusere på  panelet som er interessant, ikke ta med hele siden og "Figma" i toppen, det er bare støy.